shared_model/interfaces/query_responses/error_query_response.hpp

```diff
@@ -72,6 +60,18 @@ namespace shared_model {
        */
       virtual const ErrorMessageType &errorMessage() const = 0;
 
+      /// Error code type
+      using ErrorCodeType = uint32_t;
```

Comment:
Should not this be defined in `shared_model/interfaces/common_objects/types.hpp`?